components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/impl/WebAppAuthenticatorImpl.java

```diff
@@ -119,13 +130,13 @@ private boolean validateScopes(Message message, AccessTokenInfo tokenInfo) {
                             }
                             return true;
                         }
-                    }
-                    else {
+                    } else {
                         if (log.isDebugEnabled()) {
-                        log.debug("Scope not defined in swagger for matching resource. So consider as anonymous permission and" +
-                                "let request to");
-                            return true;
+                            log.debug(
```

Comment:
Fixed in #1581 (https://github.com/wso2/carbon-apimgt/commit/f8d3803b31f84e0c4b6ca57332dc04f72e8aec7f)